python/sglang/srt/entrypoints/engine.py

```diff
@@ -246,7 +246,16 @@ def encode(
 
     def shutdown(self):
         """Shutdown the engine"""
-        kill_process_tree(os.getpid(), include_parent=False)
```

Comment:
`include_parent` is already `False` by default, so it won't kill the parent process.
You changed it to True, and it failed many CI test. Is this still necessary?

Reply:
I see. I didn't notice the change. let me check